src/transformers/models/encodec/modeling_encodec.py

```diff
@@ -762,6 +803,24 @@ def decode(
             return (audio_values,)
         return EncodecDecoderOutput(audio_values)
 
+    def compute_mel_spectrogram(self, audio, n_fft, hop_length, n_mels=64):
```

Comment:
I've just realized that `compute_mel_spectrogram` doesn't really depend on the model, let's add `sampling_rate` in the argument list and move it to to `loss_encodec.py` ?